react/components/molecules/key-values/key-values.stories.js

```diff
@@ -12,7 +12,21 @@ storiesOf("Molecules", module)
             { key: "Company", value: "Platforme", border: "none" },
             { key: "Position", value: "Head of Software Development", border: "soft" },
             { key: "Birth date", value: "14/03/1993", border: "hard" },
-            { key: "Nationality", value: "Portuguese" }
+            { key: "Nationality", value: "Portuguese" },
+            {
+                key: "section",
+                section: [
+                    { key: "section-E-mail", value: "gcc@platforme.com" },
+                    { key: "section-Company", value: "Platforme", border: "none" },
+                    {
+                        key: "section-Position",
+                        value: "Head of Software Development",
+                        border: "soft"
+                    },
+                    { key: "section-Birth date", value: "14/03/1993", border: "hard" },
+                    { key: "section-Nationality", value: "Portuguese" }
+                ]
+            }

```

Comment:
can't we simplify this structure by simply adding separator label objects in the position where we want it to show instead of making this section like structure? Example:
```suggestion
            { separator: "Nationality" },
            { key: "E-mail", value: "gcc@platforme.com" },
            { key: "Company", value: "Platforme", border: "none" },
            { key: "Position", value: "Head of Software Development" },
            { key: "Birth Date", value: "14/03/1993" },
            { key: "nationality", value: "Portuguese" }
```

Also I'm starting to think that we could simplify even more by creating a component called `SeparatorTitle` or similar and use it in robin just before the key-values that we wanted to section. It would be simpler to implement and avoid possible regressions. What do you think is best?